src/server/display_server.cpp

```diff
@@ -176,15 +166,9 @@ struct mir::DisplayServer::Private
                 [this] { input_dispatcher->start(); },
                 [&, this] { input_dispatcher->stop(); });
 
-            auto prompt = try_but_revert_if_unwinding(
-                [this] { prompt_connector->start(); },
-                [&, this] { prompt_connector->stop(); });
-
             auto wayland = try_but_revert_if_unwinding(
                 [this] { wayland_connector->start(); },
                 [&, this] { wayland_connector->stop(); });
-
```

Comment:
I agree it is misleading, but not wrong: there is no way that `wayland_connector->stop();` can be called (as there is no more code in this scope). Specifically, `[this] { wayland_connector->start(); }` is called in the constructor, so the destructor won't run if it throws (it would be wrong everywhere else if that were not the case).

Reply:
+1 from me.